.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
Seeing that this differs per person, it should actually live in a [global .gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) and not in this repository.